README.md

```diff
@@ -44,6 +44,7 @@ Available functions:
 * `POW(expr, power)` - Return the argument raised to the specified power
 * `SIGN(expr)` - Return the sign of the argument
 * `CAST(expr as type)` - Takes an expression of any type and produces a result value of a specified type. Supported types are: "char, string, text, date, datetime, time, int, integer, decimal, boolean"
+* `BASE64_LENGTH` - Return number of bytes of decoded string
```

Comment:
Extensions contain functions that are present in DBs. `BASE64_LENGTH` seems to be very specific one and may be released as 2 separate functions: `LENGTH` and `FROM_BASE64`
Please, be sure that base64 decode is supported for all supported DBs, as doctine-extensions is a separate package which is not strictly connected with crm/commerce/platform projects